src/batched/KokkosBatched_Util.hpp

```diff
@@ -43,6 +43,12 @@ namespace KokkosBatched {
 #pragma message "KOKKOS_BATCHED_DISABLE_AVX512"
 #endif
 
+#if defined(__FMA__)
+#pragma message "KOKKOS_BATCHED_ENABLE_FMA"
```

Comment:
I put the message (I saw this from bcrs mimic sparc) because I want to see what intrinsics are used when compiling this code.